code/__DEFINES/mobs.dm

```diff
@@ -293,7 +293,7 @@
 #define DOOR_CRUSH_DAMAGE	15	//the amount of damage that airlocks deal when they crush you
 
 #define	HUNGER_FACTOR		0.1	//factor at which mob nutrition decreases
-#define	ETHEREAL_CHARGE_FACTOR	0.12 //factor at which ethereal's charge decreases
+#define	ETHEREAL_CHARGE_FACTOR	0.1 //factor at which ethereal's charge decreases
```

Comment:
```suggestion
#define	ETHEREAL_CHARGE_FACTOR	0.8 //factor at which ethereal's charge decreases
```
something like this is fine too

Reply:
> ETHEREAL_CHARGE_FACTOR **.8** You mean .08 right? Thats a typographical error?

Reply:
yeah lol. u should do the math tho and tell us how long it currently takes an ethereal to go starving from x nutrition level vs humans (and what this would do). blindly picking numbers and then having to choose a better one later is bad

Reply:
So full hunger is 100, starving is below 25, I dont know how many ticks or seconds or whatever the factor is called to drain stomach charge.  According to my calculations, it takes around 20 minutes to go from 100 (full hunger) to 25, or starvation and the red power cell. Not only do humans have more storage for energy (food), but they also lose less per life cycle (2 seconds).

Reply:
Ive heard humans have something like 350, which is about an **HOUR AND TWO THIRDS** of time before starvation.

Reply:
hunger_factor is not how much food humans lose its a multiplier on all food loss.

Reply:
Then what is humans food loss?

Reply:
One thing u gotta keep in mind is that im pretty sure human satiety rate is affected by factors like mood (i think) and their current satiety level (pretty sure)